pkg/backend/openshiftcluster.go

```diff
@@ -30,13 +30,28 @@ type openShiftClusterBackend struct {
 	newManager func(context.Context, *logrus.Entry, env.Interface, database.OpenShiftClusters, database.Gateway, encryption.AEAD, billing.Manager, *api.OpenShiftClusterDocument, *api.SubscriptionDocument) (cluster.Interface, error)
 }
 
+type clusterResult struct {
```

Comment:
operationResult seems more generic

Reply:
please also implement String() function

Reply:
Changed name to operationResult, and created a String() function to simplify things.